setup.py

```diff
@@ -48,7 +48,7 @@
     install_requires=[
         'numpy>=1.14.5',
         'torch>=1.9.0,<1.11.0',
-        'pyrouge>=0.1.3',
+        'rouge_score>=0.0.4',
```

Comment:
Doesn't HF dataset already install this as a requirement?

Reply:
Nope. Same as bert-score we need to install dependencies separately.